flink-mesos/src/main/java/org/apache/flink/mesos/runtime/clusterframework/MesosTaskManagerParameters.java

```diff
@@ -64,7 +64,7 @@
 	 */
 	@Deprecated
 	public static final ConfigOption<Integer> MESOS_RM_TASKS_MEMORY_MB =
-		key("mesos.resourcemanager.tasks.mem")
+		key("taskmanager.memory.total-process.size")
```

Comment:
I think we could not simply change the key.

Reply:
from https://github.com/apache/flink/pull/10513/files#r356585015, just need to delete the code and comments that use MEMOS_RM_TASKS_MEMORY_MB?